etsin_finder/frontend/js/components/qvain/description/issuedDateField.jsx

```diff
@@ -1,16 +1,24 @@
 import React, { Fragment } from 'react'
 import PropTypes from 'prop-types'
 import { inject, observer } from 'mobx-react'
-import { SingleDatePicker } from 'react-dates';
-import styled from 'styled-components'
 import Translate from 'react-translate-component'
+import translate from 'counterpart'
 import moment from 'moment'
+import parseDate from 'moment-parseformat'
+import DatePicker, { registerLocale } from 'react-datepicker'
+import fi from 'date-fns/locale/fi'
+import en from 'date-fns/locale/en-GB'
 import Card from '../general/card'
 import ValidationError from '../general/validationError'
 import { LabelLarge } from '../general/form'
 import { issuedDateSchema } from '../utils/formValidation';
 import DateFormats from '../utils/date'
 
+registerLocale('fi', fi)
+registerLocale('en', en)
+
+import 'react-datepicker/dist/react-datepicker.css'
```

Comment:
Import/register these globally in e.g. app.jsx so individual components don't need to do this.